app/src/main/java/org/simlar/https/UploadLogFile.java

```diff
@@ -201,7 +201,8 @@ protected PostResult doInBackground(final File... logFiles)
 				deleteFile(logFile);
 
 				try {
-					final Process p = Runtime.getRuntime().exec("logcat -d -v threadtime -f " + logFile.getAbsolutePath());
+					//noinspection UseOfProcessBuilder
```

Comment:
It is definitely better this way. I could add the RuntimeExec inspection now.
Removing the UseOfProcessBuilder inspection completely would have been possible, too.